app/controllers/users_controller.rb

```diff
@@ -93,6 +93,14 @@ def impersonate
     end
   end
 
+  def invalidate_jwt_for_all_users
+    @users = User.authorized(:edit_users).except_hidden.uniq
```

Comment:
```suggestion
    user_ids = User.authorized(:edit_users).except_hidden.ids.uniq
```

We don't need anything else except the `id`.